

- Add acpi_floppy_exit()

- Remove pointlessly out-of-lined stubs


Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/block/floppy.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff -puN drivers/block/floppy.c~add-acpi-based-floppy-controller-enumeration-fix drivers/block/floppy.c
--- 25/drivers/block/floppy.c~add-acpi-based-floppy-controller-enumeration-fix	2004-08-28 14:25:58.508532776 -0700
+++ 25-akpm/drivers/block/floppy.c	2004-08-28 14:30:24.645073872 -0700
@@ -4387,8 +4387,8 @@ static void acpi_floppy_exit(void)
 	}
 }
 #else
-static int  acpi_floppy_init(void) { return -ENODEV; }
-static void acpi_floppy_stop(void) { }
+static inline int  acpi_floppy_init(void) { return -ENODEV; }
+static inline void acpi_floppy_exit(void) { }
 #endif
 
 int __init floppy_init(void)
_
